cobalt/BUILD.gn

```diff
@@ -92,6 +92,11 @@ source_set("common") {
     "app/cobalt_switch_defaults.h",
   ]
 
+  public_deps = [
```

Comment:
It doesn't even seem necessary to have these two be public_deps, but if you feel strongly about it then exclude the deps from the cobalt target and have them come transitively